spikeinterface/curation/tests/test_sortingview_curation.py

```diff
@@ -44,44 +44,47 @@ def test_sortingview_curation():
-    assert len(sorting_curated_jot_rejected1.unit_ids) == 4
-
     # curation_link: 
-    # https://figurl.org/f?v=gs://figurl/spikesortingview-10&d=sha1://1ba03f81e62ec7cb2e3e46898830f92cdf5e026f&label=SpikeInterface%20-%20Sorting%20Summary&s={%22sortingCuration%22:%22sha1://59feb326204cf61356f1a2eb31f04d8e0177c4f1%22}
-    sha_uri = "sha1://59feb326204cf61356f1a2eb31f04d8e0177c4f1"
-    sorting_curated_sha = apply_sortingview_curation(sorting, uri_or_json=sha_uri, verbose=True)
-
-    assert len(sorting_curated_sha.unit_ids) == 9
-    assert "#8-#9" in sorting_curated_sha.unit_ids
-    assert "accept" in sorting_curated_sha.get_property_keys()
-    assert "mua" in sorting_curated_sha.get_property_keys()
-    assert "artifact" in sorting_curated_sha.get_property_keys()
-
-    sorting_curated_sha_accepted = apply_sortingview_curation(sorting, uri_or_json=sha_uri, include_labels=["accept"])
-    sorting_curated_sha_mua = apply_sortingview_curation(sorting, uri_or_json=sha_uri, exclude_labels=["mua"])
-    sorting_curated_sha_mua1 = apply_sortingview_curation(sorting, uri_or_json=sha_uri,
+    # https://figurl.org/f?v=gs://figurl/spikesortingview-10&d=sha1://bd53f6b707f8121cadc901562a89b67aec81cc81&label=SpikeInterface%20-%20Sorting%20Summary
+
+    # from curation.json
+    json_file = "sv-sorting-curation.json"
+    sorting_curated_json = apply_sortingview_curation(sorting, uri_or_json=json_file, verbose=True)
+    print(f"From JSON: {sorting_curated_json}")
+
+    assert len(sorting_curated_json.unit_ids) == 9
+    assert "#8-#9" in sorting_curated_json.unit_ids
+    assert "accept" in sorting_curated_json.get_property_keys()
+    assert "mua" in sorting_curated_json.get_property_keys()
+    assert "artifact" in sorting_curated_json.get_property_keys()
+
+    sorting_curated_json_accepted = apply_sortingview_curation(sorting, uri_or_json=json_file, include_labels=["accept"])
+    sorting_curated_json_mua = apply_sortingview_curation(sorting, uri_or_json=json_file, exclude_labels=["mua"])
+    sorting_curated_json_mua1 = apply_sortingview_curation(sorting, uri_or_json=json_file,
                                                           exclude_labels=["artifact", "mua"])
-    assert len(sorting_curated_sha_accepted.unit_ids) == 3
-    assert len(sorting_curated_sha_mua.unit_ids) == 6
-    assert len(sorting_curated_sha_mua1.unit_ids) == 5
+    assert len(sorting_curated_json_accepted.unit_ids) == 3
+    assert len(sorting_curated_json_mua.unit_ids) == 6
+    assert len(sorting_curated_json_mua1.unit_ids) == 5
+
+    # from GH
+    # curated link: https://figurl.org/f?v=gs://figurl/spikesortingview-10&d=sha1://bd53f6b707f8121cadc901562a89b67aec81cc81&label=SpikeInterface%20-%20Sorting%20Summary&s={%22sortingCuration%22:%22gh://SpikeInterface/test-sv-sorting-curation/main/test/curation-test.json%22}
+    gh_uri = "gh://SpikeInterface/spikeinterface/master/spikeinterface/curation/tests/sv-curation-test.json"
```

Comment:
Isn't this pointing this to something that does not exist yet? (this is not yet in master). Is that the reason why the testing time decreased so drastically? 

Should we still keep the commented code above? If so, could you write a longer comment explaining what it is to future onlookers.